scanners/cordova/cordova.go

```diff
@@ -237,17 +239,17 @@ func (scanner *Scanner) Options() (models.OptionNode, models.Warnings, models.Ic
 	platforms := []string{"ios", "android", "ios,android"}
 
 	if relCordovaConfigDir != "" {
-		rootOption = models.NewOption(workDirInputTitle, workDirInputEnvKey, models.TypeSelector)
+		rootOption = models.NewOption(workDirInputTitle, workDirInputSummary, workDirInputEnvKey, models.TypeSelector)
 
-		projectTypeOption := models.NewOption(platformInputTitle, platformInputEnvKey, models.TypeSelector)
+		projectTypeOption := models.NewOption(platformInputTitle, platformInputSummary, platformInputEnvKey, models.TypeSelector)
```

Comment:
Just a note:
The new variable name is `projectTypeOption` but all the attributes are `platform` something.
(platformInputTitle, platformInputSummary, platformInputEnvKey).

Maybe a `platformTypeOption` would be better. Or something like that. 🤔